monai/networks/blocks/transformerblock.py

```diff
@@ -36,15 +36,15 @@ def __init__(
         causal: bool = False,
         sequence_length: int | None = None,
         with_cross_attention: bool = False,
+        use_flash_attention: bool = False,
     ) -> None:
         """
-        Args:
-            hidden_size (int): dimension of hidden layer.
-            mlp_dim (int): dimension of feedforward layer.
-            num_heads (int): number of attention heads.
-            dropout_rate (float, optional): fraction of the input units to drop. Defaults to 0.0.
-            qkv_bias (bool, optional): apply bias term for the qkv linear layer. Defaults to False.
-            save_attn (bool, optional): to make accessible the attention matrix. Defaults to False.
+        Args: hidden_size (int): dimension of hidden layer. mlp_dim (int): dimension of feedforward layer. num_heads
```

Comment:
Looks the format here has been changed by mistake.

Reply:
Yes, it was the "fill paragraph" button that changed these. Same for the self attention block.